corona-warn-app/cwa-app-android
# ExposureHistory - high risk by multiple low risk (EXPOSUREAPP-3515)

This PR addresses Exposureapp-3515 (high risk caused by multiple low risk encounters).

Features:
- provide special high risk inside the contact diary (caused by low risks)
- ViewModel test for contact diary

todo (done):
- fix tests
- testmenu item

how to test:
- go to testmenu
- ENF v2 Calculation
- set to Increased_Risk_Due_low_risk_encounter & calculate Risk Level
- Go to the MainFragment and open the Contact Diary
- Check the Exposure Entries for the new item

Screenshot:
![image](https://user-images.githubusercontent.com/46747780/108733973-449d3c00-752f-11eb-9745-b7a004a848f0.png)
